docs/src/main/asciidoc/_attributes.adoc

```diff
@@ -55,4 +55,8 @@
 // .
 :create-app-group-id: org.acme
 // .
+// Devtools snippet's attribute and context
+:upstream:
```

Comment:
I would rather not use upstream/downstream, as we discussed. This PR is specifically about whether or not there is support for asciidoc tabs.
`:asciidoc-tabs: true` should be the default. `:asciidoc-tabs: false` can be set where tabs aren't supported, which can toggle flat behavior (which would apply to pdf also)

Reply:
@ebullient You make a good point.

I think the original intention behind the `upstream` and `downstream` attributes was to use them to conditionalize a set of product-only changes located throughout the docs source. That way we could have  a single attribute trigger a bunch of conditionals on stuff that we only need in product docs.

For the rest of product-only changes, we could achieve a similar result to the latter by using `ifeval` statements  and an attribute named, for example, `:target:` that resolves to either  `community` or `rhbq`.

IMHO, we should consider changing the attribute for this specific use case according to your suggestion.

If we want to avoid mentioning terms like _upstream_ and _downstream_ in the community source, or if we decide that the product-only stuff cannot live in the upstream source at all, we'll need to consider another substitution mechanism, such as parameterized attributes with 2 discreet sets of attribute values, one for the community docs and the other for RHBQ.

Notably, we also want to reduce the source-level difference between upstream and downstream docs altogether. Some of the product-specific stuff, like support statements can be moved to a more visible location downstream (for example to the RN, Component details or other resources). 
Even so, we'll need parameterized attributes to substitute data like builder image names and the BOM GAV, as these differ between upstream and downstream, and must be listed accurately with respect to which version of the docs they are displayed in.

Reply:
@inoxx03 @ebullient This is not only about changing the attribute. If I apply this approach with a single attribute with two values based on repository context, it will no longer be an `ifdev` condition. I would need to rewrite all the snippets to use `ifeval` conditions instead. @maxandersen @gsmet , will this change help you in this process, or the current state is more about not touching original files and finding the other way?

Reply:
i think everyones life would be easier if we could just do it *one* way and then either transform it on HTML pages with javascript or some automatic conversion when moving it to some other publishing form.

Reply:
> i think everyones life would be easier if we could just do it one way and then either transform it on HTML pages with javascript

I could agree with that but the problem is the destructive version is the one we need. We definitely lose some structure in the downstream case and I'm not sure we will be able to easily cover all the cases (and I personally don't have the time to check all the various cases we have).

If we don't go this way, I agree with @ebullient that `:asciidoc-tabs: false` would be better. It should be a simple search/replace to get this PR updated. I can do it if you don't feel like doing it.